drgn_tools/crash_net.py

```diff
@@ -0,0 +1,368 @@
+    :param prog: drgn program
+    :param pid: pid of the process.
+    :returns: None.
+    """
+
+    task = find_task(prog, int(pid))
+    if not (task):
+        print("Invalid PID.")
+        return
+
+    print(
+        "PID: "
+        + pid
+        + "    TASK: "
+        + hex(task.value_())[2:]
+        + "    CPU: "
+        + str(int(task.cpu))
+        + "    COMMAND: "
+        + task.comm.string_().decode("utf-8")
+    )
+    print(
+        "%-5s %-20s %-20s %-20s %-20s %-20s"
+        % (
+            "FD",
+            "SOCKET",
+            "SOCK",
+            "FAMILY:TYPE",
+            "SOURCE-PORT",
+            "DESTINATION-PORT",
+        )
+    )
+    for i in range(int(task.files.fdt.max_fds)):
+        try:
+            socket = SOCKET_I(task.files.fdt.fd[i].f_path.dentry.d_inode)
+            sock_family = str(
+                sockfamily_dict.get(str(int(socket.sk.__sk_common.skc_family)))
+            )
+            if (
+                str(int(socket.sk.__sk_common.skc_family))
+                not in sockfamily_dict.keys()
+            ):
+                sock_family = "AF_"
+            sock_family = sock_family[3:]
+
+            sock_type = str(socktype_dict.get(str(int(socket.type))))
+            if str(int(socket.type)) not in socktype_dict.keys():
+                sock_type = "SOCK_"
+            sock_type = sock_type[5:]
+
+            if sock_family == "INET":
+                saddr = int(socket.sk.__sk_common.skc_rcv_saddr)
+                daddr = int(socket.sk.__sk_common.skc_daddr)
+                dport = socketlib.ntohs(int(socket.sk.__sk_common.skc_dport))
+                sport = int(socket.sk.__sk_common.skc_num)
+                source_port = (
+                    str(ipaddress.ip_address(saddr).reverse_pointer[:-13])
+                    + "-"
+                    + str(sport)
+                )
+                destination_port = (
+                    ipaddress.ip_address(daddr).reverse_pointer[:-13]
+                    + "-"
+                    + str(dport)
+                )
+            else:
+                source_port = ""
+                destination_port = ""
+            print(
+                "%-5s %-20s %-20s %-20s %-20s %-20s"
+                % (
+                    str(i),
+                    hex(socket.value_())[2:],
+                    hex(socket.sk.value_())[2:],
+                    sock_family + ":" + sock_type,
+                    source_port,
+                    destination_port,
+                )
+            )
+            print(socket)
+            print(socket.sk)
+        except Exception:
+            continue
+
+
+class CrashNet(CorelensModule):
+    """
+    Corelens Module for sysinfo
+    """
+
+    name = "net"
+
+    def add_args(self, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument(
+            "-a",
+            dest="all_data",
+            action="store_true",
+            help="Displays net -a output",
```

Comment:
```suggestion
            help="Displays the ARP table",
```
That way people don't need to know what crash's `net -a` does.